sotodlib/coords/helpers.py

```diff
@@ -209,16 +210,30 @@ def get_horiz(tod, wrap=False, dets=None, timestamps=None, focal_plane=None,
         tod.wrap(wrap, output, [(0, 'dets'), (1, 'samps')])
     return output
 
-def get_wcs_kernel(proj, ra, dec, res):
+
+def get_wcs_kernel(proj, ra=None, dec=None, res=None):
     """Construct a WCS.  This fixes the projection type (e.g. CAR, TAN),
-    reference point (the special ra, dec), and resolution of a
-    pixelization, without specifying a particular grid of pixels.
+    centered with respect to a reference point (at ra,dec = (0,0)), and
+    resolution of a pixelization, without specifying a particular grid
+    of pixels.
 
     This interface is subject to change.
 
     Args:
-      proj (str): Name of the projection to use, as processed by
-        pixell.  E.g. 'car', 'cea', 'tan'.
+      proj (str): Name and resolution of projection to be used. It
+                  must adhere to the following format:
```

Comment:
The old thing is still allowed, so should still be documented.  "Either the name of a FITS projection to use (e.g. 'car', 'cea', 'tan'), in which case "res" must also be specified, or a string containing both the projection and the resolution, in this format:"...